.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: node:8
+      - image: node:12.8
```

Comment:
why not 12.18 ?

Reply:
or even better: `node:lts`
https://circleci.com/developer/images/image/cimg/node
![Bildschirmfoto 2020-09-03 um 11 13 45](https://user-images.githubusercontent.com/14888529/92095923-9661d680-edd6-11ea-8ee7-dacd4a42b52e.png)

Reply:
oh that's a typo :D i wanted to type 12.18 but missed the `1`. I will use lts 👍

Reply:
done 👍